lib/kube/proxy/auth_test.go

```diff
@@ -252,11 +258,26 @@ current-context: foo
 				},
 			},
 			assertErr: require.NoError,
+		}, {
+			desc:               "kubernetes_service, bad kube creds",
+			serviceType:        KubeService,
+			kubeconfigPath:     kubeconfigPath,
+			impersonationCheck: alwaysSucceeds,
+			assertErr: func(tt require.TestingT, err error, i ...interface{}) {
+				findErr := "failed to generate TLS config from kubeConfig. clientConfig"
+				for sc.Scan() {
+					if strings.Contains(sc.Text(), findErr) {
+						return
+					}
+				}
+				require.Fail(tt, "Failed to find error", "Failed to find error %q in the logs", findErr)
```

Comment:
Simplify? require.Fail is a bit pointless, IMO, plus the stuttering message is weird.

```suggestion
				tt.Fatalf("Failed to find error %q in the logs", findErr)
```